package.json

```diff
@@ -27,13 +27,20 @@
     "material-ui-chip-input": "^2.0.0-beta.2",
     "mongodb": "^3.5.5",
     "mongoose": "^5.9.7",
+    "multer": "^1.4.2",
     "nodemailer": "^6.4.6",
     "passport": "^0.4.1",
     "passport-facebook": "^3.0.0",
     "passport-google-oauth": "^2.0.0",
     "passport-local": "^1.0.0",
+<<<<<<< Updated upstream
```

Comment:
I can not run the latest commit due to this merge conflict

Reply:
<img width="651" alt="server_errors" src="https://user-images.githubusercontent.com/6497687/86705157-72e30100-bfca-11ea-8a55-c478c464c066.png">

Reply:
<img width="1318" alt="client_display" src="https://user-images.githubusercontent.com/6497687/86705238-868e6780-bfca-11ea-9035-54db9564f74b.png">

Reply:
Did you install node-module? If you didn't install it, please install and check it again.